app/src/test/java/org/mozilla/fenix/settings/creditcards/CreditCardItemViewHolderTest.kt

```diff
@@ -50,7 +50,7 @@ class CreditCardItemViewHolderTest {
     fun `GIVEN a new credit card item on bind THEN set the card number and expiry date text`() {
         CreditCardItemViewHolder(view, interactor).bind(creditCard)
 
-        assertEquals(creditCard.encryptedCardNumber.number, view.credit_card_number.text)
+        assertEquals(creditCard.cardNumberLast4, view.credit_card_number.text)
```

Comment:
Are we able to verify the displayed card number with the ellipsis?

Reply:
Yep, works.